schema/opentelemetry_configuration.json

```diff
@@ -11,6 +11,9 @@
         "disabled": {
             "type": ["boolean", "null"]
         },
+        "log_level": {
+            "type": ["string", "null"]
```

Comment:
Can we have an enum based on a subset of the SeverityNumber enum defined in the proto repo? https://github.com/open-telemetry/opentelemetry-proto/blob/4f69356d853029975649c3f38b06fc77d77975fc/opentelemetry/proto/logs/v1/logs.proto#L86-L110

Reply:
We could, though without a resolution on https://github.com/open-telemetry/opentelemetry-specification/issues/2039, we may be jumping ahead of the spec